packages/wells/src/client/api/wellbores.ts

```diff
@@ -32,18 +35,56 @@ export class Wellbores extends AssetsAPI {
    */
   private mapToWellbore = async (assets: Asset[]): Promise<Wellbore[]> => {
     return assets.map(asset => {
-      return <Wellbore>{
+      const wellbore = <Wellbore>{
         id: asset.id,
         name: asset.name,
-        parentId: asset.parentId,
         metadata: asset.metadata,
         trajectories: async (): Promise<Survey[]> => {
           return await this.surveys.listTrajectories(asset.id);
         },
+        parentWell: async (): Promise<Well | undefined> => {
+          return await this.getParentWell(asset);
+        },
+      };
+      wellbore.getDatum = async (): Promise<WellDatum | undefined> => {
+        const well = await wellbore.parentWell();
+        if (well) {
+          return well.datum;
+        }
+      };
+      wellbore.getWellHeadLocation = async (): Promise<
+        WellHeadLocation | undefined
+      > => {
+        const well = await wellbore.parentWell();
+        if (well) {
+          return well.wellHeadLocation;
+        }
       };
+      return wellbore;
     });
   };
 
+  /**
+   * Method for finding the parent wellbore of a given wellbore asset
+   *
+   * ```
+   * const parent = await client.wellbores.getParentWell(asset);
+   * ```
+   * @param asset
+   */
+  private getParentWell = async (asset: Asset): Promise<Well | undefined> => {
+    if (asset.parentId) {
+      let parentWell = await this.retrieve([{ id: asset.id }]);
+      while (parentWell[0].metadata && parentWell[0].metadata.type !== 'Well') {
+        if (!asset.parentId) {
+          return undefined;
+        }
+        parentWell = await this.retrieve([{ id: asset.parentId }]);
```

Comment:
nit, but would it be possible to retrieve the first item of the result here so parentWell is am actual well instead of a list of wells, and then not have to do parentWell[0] in the while and return statements? It seems a bit clearer